resources/site-packages/elementum/daemon.py

```diff
@@ -42,9 +48,6 @@
 lib = None
 last_lib = ""
 
-class GoString(Structure):
```

Comment:
Are you sure without this all the strings are properly passed between shared library and python?

Reply:
@elgatito its usage was removed in https://github.com/elgatito/plugin.video.elementum/commit/c4869e6455330cbabb668c650e889d4042b491e1 so i decided to remove it. correct me if i am wrong.